js/ui/data_grid/ui.data_grid.summary.js

```diff
@@ -276,7 +276,7 @@ gridCore.registerModule("summary", {
                 /**
                  * @name dxDataGridOptions_summary_groupItems
                  * @publicName groupItems
-                 * @type array
+                 * @type Array<Object>
```

Comment:
Array<dxDataGridOptions_summary_groupItems>

Reply:
It's some kind of compromise. We should create separate objects for all these cases - but now we suppose that all nested options of array members are fields of the inner object, and declare an array as "Array<Object>"